components/Header.tsx

```diff
@@ -78,8 +78,8 @@ const Header = ({ setSidebarOpen }: { setSidebarOpen: any }) => {
   const [osModifier, setOsModifier] = useState("")
 
   useEffect(() => {
-    if (navigator.userAgent.indexOf("Mac OS X") !== -1) setOsModifier("⌘")
-    if (navigator.userAgent.indexOf("Windows") !== -1) setOsModifier("Ctrl + ")
+        if (navigator.userAgent.indexOf("Mac OS X") !== -1) { setOsModifier("⌘") }
```

Comment:
Can we use a guard clause here instead of if/else?

Reply:
Sure